config/config.js

```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+module.exports = {
+    development: {
+        username: process.env.DB_USER,
+        password: process.env.DB_PASSWORD,
+        database: process.env.DB_NAME,
+        host: "127.0.0.1",
+        port: 5432,
+        dialect: "postgres"
```

Comment:
Make the dialect configurable
```suggestion
        dialect: process.env.DIALECT || "postgres"
```